fs_xgspon_mod.py

```diff
@@ -730,9 +750,9 @@ def parse_vlan_filter(vf):
     parse_telnet.set_defaults(func=telnet)
 
     parse_install = s.add_parser("install")
-    parse_install.add_argument("fs_onu_serial", type=parse_serial)
+    parse_install.add_argument("serial", type=parse_serial)
```

Comment:
I would like to see this backed out, just so that the argparse help is slightly clearer for users of ISPs where the distinction between the two matters. Mostly because `install` and `persist` are going to be used at the boundary where the active serial from the stick's perspective changes, so I think the extra clarity is useful.